docs/editors-guide/externally-managed-content.md

```diff
@@ -0,0 +1,32 @@
+## Automatic Mondo Quality Control Checks for Externally Managed Content
+A subset of the extensive Mondo QC checks are automatically run on the externally managed content. To limit delay due to QC failure, any content from external providers that does not pass these QC checks is not included in Mondo until the content is updated by the data providers and passes the QC checks. Reports of these QC checks will be available for review of any errors so that any needed changes can be made.
+
+### QC Checks for Externally Managed Content
+| Error | Description | SPARQL Query | 
+|:------|:------------|:-------------|
+| duplicate_scoped_synonym | The synonym from the external source is the same as a synonym already reported on the Mondo term, but the synonym scope is not the same. For example, the synonym from the source is "exact", and the synonym in Mondo is "broad".| <a href='https://robot.obolibrary.org/report_queries/duplicate_scoped_synonym' target='_blank'>duplicate scoped synonym</a> |
+| invalid_xref | A database_cross_reference in the EMC file is not in CURIE format.| <a href='https://robot.obolibrary.org/report_queries/invalid_xref' target='_blank'>invalid xref</a> |
```

Comment:
EMC is not defined in the document yet. We should have it in the first paragraph so we know that EMC is Externally Managed Content.

Reply:
done